crates/languages/src/javascript/config.toml

```diff
@@ -4,6 +4,7 @@ path_suffixes = ["js", "jsx", "mjs", "cjs"]
 # [/ ] is so we match "env node" or "/node" but not "ts-node"
 first_line_pattern = '^#!.*\b(?:[/ ]node|deno run.*--ext[= ]js)\b'
 line_comments = ["// "]
+block_comment = ["/*", "*/"]
```

Comment:
Hey, great work! You might want to add this also to the `Typescript` and `Tsx` languages.

Reply:
Thanks! I have added config for typescript and TSX as well